src/components/navigation/header/menu/icon.tsx

```diff
@@ -0,0 +1,51 @@
+import { faBell } from '@fortawesome/free-solid-svg-icons/faBell'
+import { faGraduationCap } from '@fortawesome/free-solid-svg-icons/faGraduationCap'
+import { faHandHoldingHeart } from '@fortawesome/free-solid-svg-icons/faHandHoldingHeart'
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons/faInfoCircle'
+import { faUser } from '@fortawesome/free-solid-svg-icons/faUser'
+import { faUserEdit } from '@fortawesome/free-solid-svg-icons/faUserEdit'
+import { faUserFriends } from '@fortawesome/free-solid-svg-icons/faUserFriends'
+import clsx from 'clsx'
+
+import { FaIcon } from '@/components/fa-icon'
+
+const menuIconMapping = {
+  subject: faGraduationCap,
+  about: faInfoCircle,
+  participate: faUserEdit,
+  community: faUserFriends,
+  donate: faHandHoldingHeart,
+  user: faUser,
+  login: faUser,
+  notifications: faBell,
+}
+
+export type IconIdentifier = keyof typeof menuIconMapping
+
+export interface IconProps {
```

Comment:
Am I interpreting the type right, that either `icon` or `element` need to be set? How about:

```typescript
type IconProps = { icon: IconIdentifier, alt: string } | { element: JSX.Element, alt: string }
```

Shall `element` be named `children` so that the following is possible:

```typescript
<Icon alt="...">
   <Element></Element>
</Icon>
```

Reply:
(see above).